tas-installer/internal/kubernetes/pod.go

```diff
@@ -0,0 +1,57 @@
+package kubernetes
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	MaxAttempts    = 30
+	SleepInterval  = 10
+	ErrPodNotFound = errors.New("pod not found")
+)
+
+func (kc *KubernetesClient) CheckPodStatus(namespace, podNamePrefix string) (string, error) {
```

Comment:
Maybe `GetPodStatus()` since it's not really checking, just getting the status string.